TEveEventDisplay/src/TEveMu2e_base_classes/TEveMu2eCustomHelix.h

```diff
@@ -3,27 +3,22 @@
 
```

Comment:
Having this class in TEveMu2e_base_classes suggest that you plan to use this as a base class and to make derived classes.  However the class has no virtual functions and there are no classes that inherit from it?    So what's the meaning of the name base_classes in this context?

Reply:
hi Rob, the name just means that they are fundamental classes of the TEve objects, I dont intend to have inheritance between  them, I can change the name of the directory if you want